### ISSUE TITLE
Node option is not avaiable in Integrated terminal

### ISSUE THREAD

**Issue by senguptaavijit6** [microsoft/vscode#121787]
Issue Type: <b>Bug</b>

When I'm trying to execute any command that starts with "ng" it's not executing and opening a file (C:\Users\{{user's_name}}\AppData\Roaming\npm\node_modules\@angular\cli\bin).

but when I'm executing the same command(s) with git bash it's executing very smoothly without any error or warning.

I've opened the project folder in the editor (VSCode) and not just a single file.

I've disabled all my extensions to ensure that it's not the problem with the extensions, but the problem persists. 

my current packages with version
    - windows 10 Pro x64                    20H2
    - OS build                               19042.928
    - VS code                                       1.55.2
    - node (VSCode Terminal)            14.16.1 
    - node (VSCode about)                 12.18.3
    - npm                                              7.10.0
    - @angular-devkit/architect           0.1001.7
    - @angular-devkit/build-angular   0.1001.7
    - @angular-devkit/core                10.1.7
    - @angular-devkit/schematics      10.1.7
    - @angular/cli                               10.1.7
    - @schematics/angular                 10.1.7
    - @schematics/update                    0.1001.7
    - rxjs                                                6.6.7
    - typescript                                     4.0.7

Note:- I'm facing this issue after updating the node from version 12.*.* to 14.16.1, it may be a coincidence but I thought I should mention this also to help you understand the error more efficiently.

![1](https://user-images.githubusercontent.com/71085726/115496140-a86d8800-a286-11eb-833e-47898977e785.png)
![2](https://user-images.githubusercontent.com/71085726/115496144-aa374b80-a286-11eb-9f60-db47ea13b120.png)
![3](https://user-images.githubusercontent.com/71085726/115496145-aacfe200-a286-11eb-8b81-2f5d623c611b.png)
![4](https://user-images.githubusercontent.com/71085726/115496146-aacfe200-a286-11eb-8595-6465e4fcbd95.png)
![5](https://user-images.githubusercontent.com/71085726/115496147-ab687880-a286-11eb-8281-ad64e018f105.png)
![6](https://user-images.githubusercontent.com/71085726/115496148-ac010f00-a286-11eb-9e08-3c6ec3b3cc4a.png)
![7](https://user-images.githubusercontent.com/71085726/115496149-ac99a580-a286-11eb-91a2-766fb0617351.png)
![8](https://user-images.githubusercontent.com/71085726/115496150-ac99a580-a286-11eb-88b0-06c46dc5262f.png)
![9](https://user-images.githubusercontent.com/71085726/115496153-ad323c00-a286-11eb-8cf3-604088f8f4dc.png)
![10](https://user-images.githubusercontent.com/71085726/115500972-f8048180-a28f-11eb-977c-11bb24cefafc.png)

VS Code version: Code 1.55.2 (3c4e3df9e89829dce27b7b5c24508306b151f30d, 2021-04-13T09:35:57.887Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-3220 CPU @ 3.30GHz (4 x 3293)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.97GB (3.17GB free)|
|Process Argv|--crash-reporter-id 5c04a9d4-8a13-48d6-aac0-85e7ca1867e0|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspyt653:30270858
pythonvspyt602:30291499
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry914:30276682
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
pythonvspyt639:30291489
pythontb:30283811
pythonvspyt551cf:30291415
vspre833:30267464
pythonptprofiler:30281270
vshan820cf:30276953
pythondataviewer:30285071
vscus158cf:30286554

```

</details>

**Comment by Tyriar** [comment]
You're using git bash externally and Windows PowerShell inside VS Code, did you try using git bash in VS Code?

**Comment by senguptaavijit6** [comment]
Firstly thanks a lot that you replied!

yes, I've tried Git Bash inside VS Code, and it's working fine except for one accessibility issue, I need to create multiple instances to run commands properly. like if in run "ng serve" it will freeze the terminal and I must need to create another instance using the "+" symbol right next to the option for choosing the type of terminal I want (e.g. Powershell, Git Bash etc).
and only then I can execute the other commands, 
but with the node option selected (which was automatically chosen by vs code) I have used the terminal without any issues like these.
